libnavigation-core/src/test/java/com/mapbox/navigation/core/trip/session/MapboxTripSessionTest.kt

```diff
@@ -566,17 +565,19 @@ class MapboxTripSessionTest {
     }
 
     @Test
-    fun setRoute() {
-        tripSession.setRoutes(routes, legIndex, updateReason)
+    fun setRoutes() {
+        val alternative: NavigationRoute = mockk()
+        tripSession.setRoutes(routes + alternative, legIndex, updateReason)
 
-        coVerify(exactly = 1) { navigator.setRoute(routes, legIndex) }
+        coVerify(exactly = 1) { navigator.setPrimaryRoute(routes.first(), legIndex) }
+        coVerify(exactly = 1) { navigator.setAlternativeRoutes(listOf(alternative)) }
     }
 
     @Test
     fun setRoute_nullable() {
         tripSession.setRoutes(emptyList(), 0, RoutesExtra.ROUTES_UPDATE_REASON_CLEAN_UP)
 
-        coVerify(exactly = 1) { navigator.setRoute(emptyList(), 0) }
+        coVerify(exactly = 1) { navigator.setPrimaryRoute(null, 0) }
```

Comment:
maybe?
```suggestion
        coVerify(exactly = 1) { navigator.setPrimaryRoute(null, 0) }
        coVerify(exactly = 0) { navigator.setAlternativeRoutes(any()) }
```

Reply:
The workaround that this test would verify isn't needed anymore :tada: